### DISCUSSION TITLE
Restrict Asset Upload File Extensions

### DISCUSSION THREAD

**Discussion by kilsicles** [statamic/cms#11120] [General]
Hey everybody,

is there a way to restrict the upload of files in the Asset/Downloads section to certain file extensions (e.g. ".pdf")?

Thanks in advance!

**Comment by jasonvarga** [comment]
Yes, edit the container's settings and add a validation rule.

![CleanShot 2024-11-13 at 15 05 14](https://github.com/user-attachments/assets/3ae2367f-847f-4c8e-a54e-cb43be6cfa0e)

**Reply by kilsicles** [reply]
Thanks, maybe add a description? Couldn't figure out the syntax from the documentation...
